unattended_installer/cert_tool/certFunctions.sh

```diff
@@ -9,10 +9,10 @@
 
 function cert_cleanFiles() {
 
-    eval "rm -f /tmp/wazuh-certificates/*.csr ${debug}"
-    eval "rm -f /tmp/wazuh-certificates/*.srl ${debug}"
-    eval "rm -f /tmp/wazuh-certificates/*.conf ${debug}"
-    eval "rm -f /tmp/wazuh-certificates/admin-key-temp.pem ${debug}"
+    eval "rm -f ${tmp_path}/*.csr ${debug}"
```

Comment:
Please use a more descriptive variable name, for example, `cert_tmp_path` to avoid possible confusion.